gallery/src/styles/NxList/NxListBulletedExample.html

```diff
@@ -26,7 +26,10 @@ <h3 class="nx-h3">
         <span class="nx-list__text">Item 3</span>
         <ul class="nx-list nx-list--bulleted">
           <li class="nx-list__item">
-            <span class="nx-list__text">Item 1</span>
+            <span class="nx-list__text">
+              Item 1 this list items is also long enough that it wraps, or at least it should if I have typed enough
+              text, how much wood would a woodchuck chuck
```

Comment:
Should this wrap at the paragraph width?

Reply:
> Should this wrap at the paragraph width?

I don't know. @derrek?

Reply:
> > Should this wrap at the paragraph width?
> 
> I don't know. @derrek?

Per discussion on Slack wrapping and truncation happen at 950px, which hardly seems worth putting in a variable since it's beyond the max-width of most elements